README.md

```diff
@@ -17,9 +17,9 @@ cd CMSSW_12_1_0_pre4/src
 cmsenv
 git cms-init -q
 git cms-merge-topic yeckang:mapping_update_v0.5 # Will be updated
-git clone git@github.com:gem-sw/gemsw.git
+git cms-merge-topic jshlee:GEMGeometryBuilder-bugfix-CMSSW_12_1_0_pre3 
```

Comment:
change to CMSSW_12_1_0_pre4, then this line is not needed

Reply:
Okay